src/main/java/homework10/MyHashMap.java

```diff
@@ -0,0 +1,169 @@
+package homework10;
+
+import java.util.Arrays;
+import java.util.Map;
+
+public class MyHashMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] arrayOfNodes;
+    private int size;
+
+    public V get(K key) {
+        if (getNode(key)== null) {
```

Comment:
why we return null, if key = null?

Reply:
we dont. return null will be if result of getNode(key) == null.

Reply:
ok, but better write:
Node temp = getNode(key);
if(temp != null){
return temp.value;
return null;

Reply:
done!